dev/tests/static/testsuite/Magento/Test/Php/_files/phpcpd/blacklist/common.txt

```diff
@@ -218,3 +218,4 @@ Magento/InventoryAdminUi/Controller/Adminhtml
 Magento/Newsletter/Model/Queue
 Magento/Framework/Mail/Template
 Magento/CheckoutAgreements/Model/Checkout/Plugin
+Magento/Developer/Test/Unit/Console/Command/DevTestsRunCommandTest.php
```

Comment:
Are you sure we also need to blacklist the file for PHP CD ?

Reply:
No we don't, just forgot to rollback the file change. One sec.